extensions/wikia/InfoboxBuilder/includes/lua/InfoboxBuilderView.lua

```diff
@@ -258,19 +258,19 @@ function InfoboxBuilderView.render( input, vars )
 		:attr( 'cellpadding', '0' )
 		:addClass( 'infobox-table' )
 
-	-- Iterate over each field and render a row
+	-- Iterate over each field and render a row if a value is not empty
```

Comment:
Three don't have `isempty` checks, just checking, but should they, or are the split and custom fields allowed with empty values?